src/server/scripts/Northrend/IcecrownCitadel/instance_icecrown_citadel.cpp

```diff
@@ -585,6 +585,10 @@ class instance_icecrown_citadel : public InstanceMapScript
                         else if (GetBossState(DATA_ROTFACE) == DONE)
                             HandleGameObject(PutricideGateGUIDs[1], false, go);
                         break;
+                    case GO_SCIENTIST_ENTRANCE:
+                        PutricideDoorGUID = go->GetGUID();
+                        if (GetData(DATA_PUTRICIDE_TRAP) == DONE)
+                            HandleGameObject(PutricideDoorGUID, false, go);
```

Comment:
this will produce problems, as door system will keep them closed even if handlegameobject is called